src/v2/coins/abstractUtxoCoin.js

```diff
@@ -995,10 +995,37 @@ class AbstractUtxoCoin extends BaseCoin {
     }).call(this);
   }
 
+
+  /**
+   * Helper function for recover()
+   * This transforms the txInfo from recover into the format that offline-signing-tool expects
+   * @param txInfo
+   * @param txHex
+   * @returns {{txHex: *, txInfo: {unspents: *}, feeInfo: {}, coin: void}}
+   */
+  formatForOfflineVault(txInfo, txHex) {
+    const response = {
+      txHex,
+      txInfo: {
+        unspents: txInfo.inputs
+      },
+      feeInfo: {},
+      coin: this.getChain()
+    };
+    delete txInfo.inputs;
```

Comment:
Why mutate the input object here?

Reply:
oh whoops i dont need that anymore, i'll remove it